.github/workflows/changelog.yml

```diff
@@ -1,6 +1,9 @@
 name: Changelog
 
 on:
+  workflow_dispatch:
```

Comment:
This means you can manually trigger this workflow, which sometimes I want to be able to do.